signers/signer-solana/src/config.ts

```diff
@@ -0,0 +1,13 @@
+type Config = {
+  customRPC: string;
+};
+
+let config: Config | undefined;
+
+export function setConfig(c: { customRPC: string }) {
+  config = c;
+}
+
+export function getConfig() {
```

Comment:
you can accept a param here and also return a proper type and infer the type using generics.

Reply:
I don't get why a getter function should accept a param.

Reply:
```js

function getConfig(name){
// ...
}

const rpc = getConfig("customRPC"); // result: string | undefined

// without param
const rpc = getConfig()?.customRPC;
```

Reply:
And for consistency, we implemented same thing in other places like `widget/embedded/src/utils/configs.ts`

Reply:
Fixed